### PR TITLE
Code pattern mods

### PR DESCRIPTION
**Pull request by ash7594**
Modifying the README to follow code pattern guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,102 @@
...
+This code pattern is for developers looking to start building blockchain applications with Hyperledger Composer. When the reader has completed this code pattern, they will understand how to:
+* Create basic business network using Hyperledger Composer
+* Deploy the network to IBM Blockchain Starter Plan manually
+* Build a Node.js web application to interact with the blockchain network using Composer API
+
 The tutorial instructions will run along side the demo once deployed, you can also view them [here](apps/vehicle-manufacture/tutorial.md)
 
-[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://console.bluemix.net/devops/setup/deploy/?repository=https%3A//github.com/ibm-blockchain/vehicle-manufacture&branch=master&env_id=ibm%3Ayp%3Aus-south&deploy-region=ibm%3Ayp%3Aus-south)
+# Architecture Flow
+![Architecture Flow](https://github.com/ash7594/vehicle-manufacture/blob/code-pattern-mods/docs/doc-images/arch-flow.png?raw=true)
```
Comment (awjh-ibm) [inline README.md]: This should point to the newly added diagram in this repo rather than a link to the fork

File: README.md [hunk 2]
```diff
@@ -1,47 +1,102 @@
-# Vehicle Manufacture Tutorial
-This is the tutorial guide for the vehicle manufacture demo. The tutorial can be run as an application to help users run through the demo, providing information on what to do as well as why blockchain is useful for the task.
+# Vehicle Manufacture with Blockchain
+Imagine you are Tesla’s Elon Musk, and have just launched the Model 3 to the public. Hundreds of thousands of car orders are pouring in and you need a way to manage the manufacturing and assembly process of these orders in an automated fashion. Interestingly, this is a perfect use-case for blockchain where we could not only keep track of the transactions securely, but also maintain transparency between all the shareholders related to the purchase.
+
+In this code pattern, we will create a vehicle manufacturing program with blockchain using Hyperledger Composer API, and demonstrate it through a Node.js web application. The application showcases the scenario of buying and manufacturing a vehicle.
```
Comment (awjh-ibm) [inline README.md]: No need for API Hyperledger Composer is valid

File: README.md [hunk 3]
```diff
@@ -1,47 +1,102 @@
...
+
+As the vehicle is assembled, components like the chassis and interior will be built or installed, and the blockchain assets will be updated. Finally, the vehicle identification number, VIN, can be automatically assigned in accordance with a smart contract, without manual regulator approval, retaining regulatory oversight.
+
+This code pattern is for developers looking to start building blockchain applications with Hyperledger Composer. When the reader has completed this code pattern, they will understand how to:
+* Create basic business network using Hyperledger Composer
+* Deploy the network to IBM Blockchain Starter Plan manually
+* Build a Node.js web application to interact with the blockchain network using Composer API
```
Comment (awjh-ibm) [inline README.md]: No need for API

**Review by awjh-ibm — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,47 +1,102 @@
-# Vehicle Manufacture Tutorial
-This is the tutorial guide for the vehicle manufacture demo. The tutorial can be run as an application to help users run through the demo, providing information on what to do as well as why blockchain is useful for the task.
+# Vehicle Manufacture with Blockchain
+Imagine you are Tesla’s Elon Musk, and have just launched the Model 3 to the public. Hundreds of thousands of car orders are pouring in and you need a way to manage the manufacturing and assembly process of these orders in an automated fashion. Interestingly, this is a perfect use-case for blockchain where we could not only keep track of the transactions securely, but also maintain transparency between all the shareholders related to the purchase.
```
Comment (awjh-ibm) [inline README.md]: No entirely sure that this covers the main use case of regulatory body oversight of manufacturers

Comment (ash7594) [reply README.md]: I see, any suggestions how I could modify this?

File: README.md (hunk 2 above)
Comment (ash7594) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (ash7594) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (ash7594) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (awjh-ibm) [reply README.md]: I think a brief statement about imagining you are a generic manufacturer who has to conform to certain regulatory standards and thus keep a record of all your business dealings to ensure that you conform to the regulator's standards and are prepared for audit. Blockchain means that these regulatory rules are embodied by code in a smart contract and thus you can ensure that by recording your dealings using that smart contract in the blockchain you are meeting the requirements laid out and the blockchain provides a full record for audit.

Comment (ash7594) [reply README.md]: Modified to use the above statements.

**Review by awjh-ibm — CHANGES_REQUESTED** [review]
Really like the wording and story any chance we could make the reference more generic rather than name dropping Tesla and Musk?

**Review by awjh-ibm — APPROVED** [review]

**Pull request merged**